assets/benchmarks/shaders/Benchmark.hlsli

```diff
@@ -20,16 +20,24 @@ struct SceneData
     float4x4 CameraViewProjectionMatrix; // Camera's view projection matrix.
     float4   LightPosition;              // Light's position.
     float4   EyePosition;                // Eye (camera) position.
-    float3   Color;                      // Color for the current scene.
 };
 
-ConstantBuffer<SceneData> Scene : register(b0);
-Texture2D                 AlbedoTexture         : register(t1);
-SamplerState              AlbedoSampler         : register(s2);
-Texture2D                 NormalMap             : register(t3);
-SamplerState              NormalMapSampler      : register(s4);
-Texture2D                 MetalRoughness        : register(t5);
-SamplerState              MetalRoughnessSampler : register(s6);
+struct ColorParams
```

Comment:
better to add `float unused;` at the end, the compiler will most likely add padding to make sure the element is 16bytes aligned anyways.